docs/experiments/multiple-focus-metavariables.md

```diff
@@ -6,22 +6,20 @@ description: "With this rule, Semgrep matches all pieces of code captured by the
 
 # Using multiple focus metavariables using set union semantics
 
-Semgrep matches all pieces of code captured by focus metavariables when you specify them in a rule. Include the metavariables that you want to focus on by specifying each in a YAML list. 
+Semgrep matches all pieces of code captured by focus metavariables when you specify them in a rule. Specify the metavariables you want to focus on in a YAML list format.
 
 :::info
-This feature is using `focus-metavariable`, see [`focus-metavariable`](/writing-rules/rule-syntax/#focus-metavariable) documentation for more information.  
+This feature is using `focus-metavariable`, see [`focus-metavariable`](/writing-rules/rule-syntax/#focus-metavariable) documentation for more information.
 :::
 
 There are two ways in which you can include multiple focus metavariables:
 
-- **Set union**: Experimental feature described below in section [Set union](#set-union). This syntax matches metavariables regardless of their position in code.
-- **Set intersection**: Only matches the overlapping region of all the focused code. For more information, see [Including more focus metavariables using set intersection semantics](writing-rules/rule-syntax/#including-more-focus-metavariables-using-set-intersection-semantics).
+- **Set union**: Experimental feature described below in the section [Set union](#set-union). This syntax matches metavariables regardless of their position in the code.
```

Comment:
I am not sure a user would understand what "This syntax matches metavariables regardless of their position in the code" really means. I am not sure I understand it myself. What this feature does is to return the union of all matches of the specified metavariables.

Reply:
```suggestion
- **Set union**: Experimental feature described below in the section [Set union](#set-union). This syntax returns the union of all matches of the specified metavariables.
```